src/ResourceManager/DataFactories/Commands.DataFactoryV2/help/Stop-AzureRmDataFactoryV2PipelineRun.md

```diff
@@ -0,0 +1,221 @@
+---
+external help file: Microsoft.Azure.Commands.DataFactoryV2.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Stop-AzureRmDataFactoryV2PipelineRun
+
+## SYNOPSIS
+Stops a pieline run in a data factory.
+
+## SYNTAX
+
+### ByFactoryName (Default)
+```
+Stop-AzureRmDataFactoryV2PipelineRun [-PipelineRunId] <String> [-Force] [-PassThru]
+ [-ResourceGroupName] <String> [-DataFactoryName] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### ByInputObject
+```
+Stop-AzureRmDataFactoryV2PipelineRun [-InputObject] <PSPipelineRun> [-Force] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByFactoryObject
+```
+Stop-AzureRmDataFactoryV2PipelineRun [-PipelineRunId] <String> [-Force] [-PassThru]
+ [-DataFactory] <PSDataFactory> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Stop-AzureRmDataFactoryV2PipelineRun** cmdlet stops a pipeline run in a data factory specified with the pieline run ID.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Stop-AzureRmDataFactoryV2PipelineRun -ResourceGroupName "ADF" -DataFactoryName "WikiADF" -PipelineRunId b9730a13-aa12-4926-a8b3-8e3a974ab0bd
```

Comment:
Can you add an example for each parameter set?  Also, make sure this isn't a real DataFactory/PipelineRunId.

Reply:
Our other cmdlets also only have 1, at most 2 examples - at some point we will have vendors to write more detailed descriptions in Powersehll docs and add more examples.

Reply:
Okay, as long as there are definite plans in the future for improving documentation.